tests/test_rooms/test_room_analytics.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+from tests.base import BaseTestCase
+from fixtures.token.token_fixture import (admin_api_token)
+from fixtures.room.room_analytics_fixtures import (
+    get_least_used_room_per_week_query,
+    get_least_used_room_per_week_response,
+    get_least_used_room_without_event_query,
+    get_least_used_room_without_event_response
+
+)
+
+
+class QueryRoomsAnalytics(BaseTestCase):
+
+    api_headers = {"Authorization": "Bearer" + " " + admin_api_token}
```

Comment:
Since we changed all headers to use the variable 'headers' and noy 'api_header' all across the application, it would make since to just use a uniform name  for consitency even as we make plans to collect them and reuse them instead of declaring them in each unit test to avoid duplication,